File: C/1_calculadora_simples.c
#include <stdio.h>

int main(){
  float num1, num2;
  float soma, sub, mult, div;

  printf("-- Calculadora Simples --:\n");
  
  printf("Digite o número 1: ");
  scanf("%f", &num1);
  printf("Digite o número 2: ");
  scanf("%f", &num2);

  soma = num1 + num2;
  sub = num1 - num2;
  mult = num1 * num2;
  div = num1 / num2;

  printf("%f + %f = %f\n", num1, num2, soma);
  printf("%f - %f = %f\n", num1, num2, sub);
  printf("%f * %f = %f\n", num1, num2, mult);
  printf("%f / %f = %f\n", num1, num2, div);

  return 0;
}

File: Haskell/1_calculadora_simples.hs
main = do

putStrLn "Digite o primeiro número:"
input1 <- getLine

putStrLn "Digite o segundo número:"
input2 <- getLine

let num1 = read input1 :: Double
    num2 = read input2 :: Double
    addition = num1 + num2
    subtraction = num1 - num2
    multiplication = num1 * num2
    division = if num1 / num2
    
putStrLn $ show num1 ++ " + " ++ show num2 ++ " = " ++ show addition
putStrLn $ show num1 ++ " - " ++ show num2 ++ " = " ++ show subtraction
putStrLn $ show num1 ++ " * " ++ show num2 ++ " = " ++ show multiplication
putStrLn $ show num1 ++ " / " ++ show num2 ++ " = " ++ show division